From a9c488781bf1646bfe9dc05e2c7a20398a146fe8 Mon Sep 17 00:00:00 2001
From: Andrew LaFrance <robert.a.lafrance5.civ@us.navy.mil>
Date: Thu, 6 Mar 2025 14:49:24 +0000
Subject: [PATCH 04/29] SDK: Update SDK Version Number [2/4]

**Description:** Update SDK Version Number

**Notes:** Automated by script in SDK/config

**Testing Performed:** Visually inspected changes, did full build no issues
---
 SDK/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/SDK/CMakeLists.txt b/SDK/CMakeLists.txt
index 47d9d9a7..20a67c33 100644
--- a/SDK/CMakeLists.txt
+++ b/SDK/CMakeLists.txt
@@ -2,9 +2,9 @@ project(SIMDIS_SDK)
 
 # Configure various version fields
 set(SIMDIS_SDK_MAJOR_VERSION 1)
-set(SIMDIS_SDK_MINOR_VERSION 22)
+set(SIMDIS_SDK_MINOR_VERSION 23)
 set(SIMDIS_SDK_REVISION_VERSION 0)
-set(SIMDIS_SDK_SOVERSION 22)
+set(SIMDIS_SDK_SOVERSION 23)
 set(SIMDIS_SDK_VERSION_STRING ${SIMDIS_SDK_MAJOR_VERSION}.${SIMDIS_SDK_MINOR_VERSION}.${SIMDIS_SDK_REVISION_VERSION} CACHE INTERNAL "SIMDIS SDK Version String" FORCE)
 
 # Give users a reasonable error message for minimum CXX version (C++20)
-- 
2.47.1.windows.2

